srcpkgs/lazarus-lcl-gtk2/template

```diff
@@ -0,0 +1,44 @@
+# Template file for 'lazarus-lcl-gtk2'
+pkgname=lazarus-lcl-gtk2
+version=1.8.4
+revision=1
+wrksrc=lazarus
+#Dependencies only exist on x86_64, x86_64-musl, i686, i686-musl, and it does not build with musl.
+only_for_archs="x86_64 i686"
+build_style=gnu-makefile
+makedepends="fpc gtk+-devel gdk-pixbuf-devel"
+depends="lazarus"
+short_desc="LCL units for gtk2"
+maintainer="mobinmob <mobinmob@disroot.org>"
+license="modified LGPL-2.0-or-later"
+homepage="https://www.lazarus-ide.org/"
+distfiles="https://sourceforge.net/projects/lazarus/files/Lazarus%20Zip%20_%20GZip/Lazarus%20${version}/lazarus-${version}.tar.gz"
+checksum=5a9801ac341a322ad1fa104d373f154d8ab72ba0add1693d0dcc8d936e6c10e9
+nopie=yes
```

Comment:
What is the reason for nopie ? please comment on it as pie is a security feature we take very seriously

Reply:
Νο build problems, must be a mistake on my part, will remove :)